exercises/03-keywords_and_variables.md

```diff
@@ -116,21 +127,34 @@ ${VARIABLE_NAME} =    variable value
 ```
 
 The `=` is optional and can be removed if you don't want to use it. You still need to have two or more
-(preferably at least four) spaces between `${VARIABLE_NAME}` and `variable value`. Also, variable names are completely
-case-insensitive. Robot also ignores spaces and `_` signs. This means that to robot these variables names:
-`${MYVARIABLE}`, `${MyVariable}`, `${MY VARIABLE}`, `${My_variable}`, and `${myvariable}` are actually all
-referring to the same variable. The recommended best practice is to define variables in the `*** Variables ***`
+(preferably at least four) spaces between `${VARIABLE_NAME}` and `variable value`.
```

Comment:
Generally, [How to Write Good Test Cases](https://github.com/robotframework/HowToWriteGoodTestCases/blob/master/HowToWriteGoodTestCases.rst#variables) recommends writing variables as well in natural syntax rather than how one would write them with a programming language. I think we should keep the same here.
```suggestion
(preferably at least four) spaces between `${VARIABLE NAME}` and `variable value`.
```